.github/workflows/ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           - fedora-34-x64
           - opensuse-leap
           - rhel8-x64
+          - raspbian-bullseye
```

Comment:
Question: Does the raspberry Pi need to be on it's own CI to run `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`?

Reply:
The action actually already have the multiarch docker running. Checkout at ci.yml:ln42.

Reply:
You're right, my bad! 😅